database_schemas/cassandra/benchflow.cql

```diff
@@ -401,4 +449,15 @@ CREATE TABLE trial_byte_size (
   experiment_id text,
   trial_id text,
   PRIMARY KEY (experiment_id, trial_id)
+);
+
+CREATE TABLE trial_IO (
+  reads bigint,
+  writes bigint,
+  total bigint,
+  experiment_id text,
+  trial_id text,
+  container_id text,
+  device text,
```

Comment:
@VincenzoFerme the io_data_ids from io_data now refer to single collected points, like in environment_data , so no point storing it since the trial analyser now finds the max for all values per device.